doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,65 @@
+[[renaming-ha]]
+== Renaming Database-Only Appliances
+
+Renaming database-only appliances in a high availability setup must be performed in a specific order to maintain database replication.
+
+//Are all steps in the appliance console menu?
+
+. Confirm and note the hostnames of the active database (primary) appliance and the standby database appliance in the appliance console. 
++
+You can verify this from the appliance summary screen of a database-only appliance, where the status for *Local Database Server* shows _primary_ or _standby_, for example:
+//Is there an easier way than logging into the appliance console of one appliance and verifying if "Local Database Server" says "running (primary)" or "running (standby)"?
++
+----
+...
+Local Database Server:   running (primary)
+...
+----
++
+. Stop the failover monitor on the non-database {product-title_short} appliances:
+.. In the appliance console menu, select *Configure Application Database Failover Monitor*. 
+.. Select *Stop Database Failover Monitor*.
+. Reset the standby by deleting the database:
+//Configure Database Replication > Configure Server as Standby? Then ??
++
+----
+# COMMAND
+----
++
+. Stop `evmserverd` on each {product-title_short} (non-database) appliance:
++
+----
+# systemctl stop evmserverd
+----
++
+. Rename the database-only appliance and restart that appliance:
+.. In the appliance console, configure the new hostname or IP address from the *Configure Network* option.
+.. Restart the renamed appliance.
+//Rename, then restart the standby if desired now?
```

Comment:
Sure? I'm not sure if this is supposed to talk about changing the hostname on the primary database or standby or both?

Also, with that said "renaming" is a very strange way to reference changing a VMs hostname. Maybe we could use different language for that?

Reply:
I'll check with the bug reporter (Felix) on this and get back to you, and will confirm that when we're talking about 'renaming an appliance' we are actually talking about 'changing a VM's hostname'.

Reply:
I've edited all the 'renaming' appliances phrasing to 'changing the hostname'